src/OrchardCore.Modules/OrchardCore.Alias/Models/AliasPartExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Alias.Indexes;
+using YesSql;
+
+namespace OrchardCore.Alias.Models
+{
+    internal static class AliasPartExtensions
```

Comment:
I'm not sure these should be internal. 
If you need to swap the driver out (which is supported), for another reason - i.e. not validation, then you're going to have to copy these extensions to the project, for no good reason - or you write your own which are wrong ;)